### PR TITLE
Blog content formatting and category in the articles list

### PR DESCRIPTION
**Pull request by lazydevorg**
The blog article's body has been changed as following:
* Blog title size
* Paragraph bottom margin
* Main image bottom margin
* Image caption top margin

The articles' list contains the category for every article.

### CODE REVIEW

File: styles/_blog-article.less [hunk 1]
```diff
@@ -42,10 +43,20 @@
   line-height: 1.4em;
 
   h1, h2, h3, h4, h5 {
-    font-weight: 800;
+    font-weight: 900;
```
Comment (asavin) [inline styles/_blog-article.less]: not going to work, we only have 800

File: styles/_blog-article.less [hunk 2]
```diff
@@ -194,9 +221,23 @@
...
+    margin-bottom: 40px;
+  }
+
   .blog-article__body {
     display: inline-block;
     max-width: 675px;
+
+    h2, h3 {
+      font-size: 30px;
+      font-weight: 900;
```
Comment (asavin) [inline styles/_blog-article.less]: same here, we only have 800

File: styles/_fonts.less [hunk 3]
```diff
@@ -83,7 +83,7 @@
 }
```
Comment (asavin) [inline styles/_fonts.less]: it would be good to define M2 typography style here

Comment (lazydevorg) [inline styles/_fonts.less]: @asavin: done!

File: styles/_blog-article.less [hunk 4]
```diff
@@ -181,6 +194,20 @@
...
+@media screen and (min-width: 750px) {
+  .blog-article__main-image {
+    margin-bottom: 30px;
+  }
+
+  .blog-article__body {
+    h2, h3 {
+      font-size: 30px;
+      line-height: 1.17;
+      font-weight: 900;
```
Comment (asavin) [inline styles/_blog-article.less]: one more 900 which is not going to work - use 800 instead

**Pull request merged**